### PR TITLE
workshop updates

### PR DESCRIPTION
**Pull request by afeld**
* Noted that a laptop is required
* Added Jenn Schiffer's photo, title, and description
* Added the Microsoft title, description, and additional names

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -296,10 +296,11 @@ <h5 class="h-header schedule-speaker__name">Justin Searls</h5>
                                     </div>
                                 </li>
                                 <li class="schedule-session">
-                                    <h3 class="h-header h-trimmed h-schedule-session">Glitch</h3>
+                                    <h3 class="h-header h-trimmed h-schedule-session">Going Front-end to Full-stack with Glitch</h3>
+                                    <img src="img/speakers/Jenn-Schiffer.jpg" alt="Jenn Schiffer" class="schedule-speaker__photo">
```
Comment (afeld) [inline index.html:300]: She doesn't have a profile in OpenCFP, hence including the image in the repository.

Comment (jakefolio) [reply index.html:300]: Any way you could size the image down to 250x250?

Comment (afeld) [reply index.html:300]: Good call - done!

**Comment by afeld** [comment]
Merging now, but happy to make any other needed adjustments in a follow-up pull request.

**Pull request merged**